commerce-test-util/src/main/java/com/liferay/commerce/test/util/CommerceTestUtil.java

```diff
@@ -103,7 +103,36 @@ public static CPDefinitionInventory addBackOrderCPDefinitionInventory(
 			StringPool.BLANK, false, false, 0, true, 1, 10000, StringPool.BLANK,
 			0, serviceContext);
 	}
+	
+	public static CommerceOrderItem addCommerceOrderItem(
+			long commerceOrderId, long cpInstanceId, int quantity, User user, Group group)
+		throws Exception {
+
+		CommerceOrder commerceOrder =
+			CommerceOrderLocalServiceUtil.getCommerceOrder(commerceOrderId);
 
+		if (commerceOrder.getCommerceCurrency() == null) {
+			CommerceCurrency commerceCurrency =
+				CommerceCurrencyTestUtil.addCommerceCurrency(
+					commerceOrder.getGroupId());
+
+			commerceOrder.setCommerceCurrencyId(
+				commerceCurrency.getCommerceCurrencyId());
+
+			CommerceOrderLocalServiceUtil.updateCommerceOrder(commerceOrder);
+		}
+
+		CommerceContext commerceContext = new TestCommerceContext(
+			commerceOrder.getCommerceCurrency(), 
+			user, 
+			group, 
+			commerceOrder.getCommerceAccount(),
+			commerceOrder);
```

Comment:
Put as many parameters on a single line as fit in 80 columns. Source formatter should flag anything like this that doesn't conform to standards. (Same with the other whitespace issues in this PR, I think.) So in general, try to be sure to run `gw formatSource` before sending a PR (and you could follow that with `gw checkSourceFormatting` to make sure you got everything).